File: 이건희/level1/숫자 문자열과 영단어.py
def solution(s):
    words = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
    for i, num in enumerate(words):
        s = s.replace(num, str(i))
    return int(s)

File: 김찬우/프로그래머스 /Lv.1/숫자 문자열과 영단어.java
class Solution {
    public int solution(String s) {
        int answer = 0;
        String result ="";
        String[] strArr = {"zero","one","two","three","four","five","six","seven","eight","nine"};
        
     
        for(int j=0; j<strArr.length; j++){

           s = s.replace(strArr[j],String.valueOf(j));


        }
        
        return Integer.valueOf(s);
    }
}
